miso-web/src/it/java/uk/ac/bbsrc/tgac/miso/webapp/integrationtest/PoolPageIT.java

```diff
@@ -277,6 +279,41 @@ public void testRemoveDilutions() {
     assertEquals(0, includedTable2.countRows());
   }
 
+  @Test
+  public void testDilutionTableWarnings() {
+    testDilutionTableWarning(801L, "(NO INDEX)");
+    testDilutionTableWarning(802L, "(NEAR-DUPLICATE INDEX)");
+    testDilutionTableWarning(803L, "(DUPLICATE INDEX)");
+  }
+
+  public void testDilutionTableWarning(long id, String warning) {
+    PoolPage page = PoolPage.getForEdit(getDriver(), getBaseUrl(), id);
+    DataTable table = page.getTable(PoolTableWrapperId.INCLUDED_DILUTIONS);
+    assertTrue("Dilution table fails to show '" + warning + "' warning", table.doesColumnContainSubstring(Columns.INDICES, warning));
+  }
+
+  @Test
+  public void testHeaderWarnings() {
+    testHeaderWarning(801L, "This pool contains at least one library with no index!");
+    testHeaderWarning(802L, "This pool contains near-duplicate indices!");
+    testHeaderWarning(803L, "This pool contains duplicate indices!");
+    testHeaderWarning(804L, "This pool contains at least one low quality library!");
+  }
+
+  public void testHeaderWarning(long id, String warning) {
+    PoolPage page = PoolPage.getForEdit(getDriver(), getBaseUrl(), id);
+    assertTrue("Page fails to show '" + warning + "' warning", page.getField(Field.WARNINGS).contains(warning));
+  }
+
+  @Test
+  public void testNoWarnings() {
+    PoolPage page = PoolPage.getForEdit(getDriver(), getBaseUrl(), 120001L);
+    DataTable table = page.getTable(PoolTableWrapperId.INCLUDED_DILUTIONS);
+    assertFalse(table.doesColumnContainSubstring(Columns.INDICES, "(NO INDEX)"));
+    assertFalse(table.doesColumnContainSubstring(Columns.INDICES, "(NEAR-DUPLICATE INDEX)"));
```

Comment:
This should be "INDICES" instead of "INDEX"

Reply:
The Dilutions Included table displays warnings that refer to a singular index, so I think "INDEX" is correct here. It should, however, also be "INDEX" on the line below.